test/integration/imagechange_buildtrigger_test.go

```diff
@@ -465,6 +465,8 @@ func TestMultipleImageChangeBuildTriggers(t *testing.T) {
 	}
 	defer watch2.Stop()
```

Comment:
not sure what you mean?  one is watching the build and one is watching the buildconfigs